Dockerfile

```diff
@@ -1,20 +1,21 @@
 # Stage: base image
-FROM node:22.14-bookworm-slim as base
-
-ARG BUILD_NUMBER
-ARG GIT_REF
-ARG GIT_BRANCH
+FROM node:22-alpine AS base
 
 LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
 
+RUN apk add --no-cache tzdata
 ENV TZ=Europe/London
 RUN ln -snf "/usr/share/zoneinfo/$TZ" /etc/localtime && echo "$TZ" > /etc/timezone
 
 RUN addgroup --gid 2000 --system appgroup && \
-        adduser --uid 2000 --system appuser --gid 2000
+    adduser --uid 2000 --system appuser --ingroup appgroup
 
 WORKDIR /app
 
+ARG BUILD_NUMBER
+ARG GIT_REF
+ARG GIT_BRANCH
+
 # Cache breaking and ensure required build / git args defined
```

Comment:
Not necessarily related to this PR, but maybe we need to update this line, as I believe the build number is no longer used for cache-busting as of https://github.com/ministryofjustice/hmpps-template-typescript/pull/377